src/components/ChangePlan/Checkout/Discounts/Discounts.test.js

```diff
@@ -0,0 +1,209 @@
+import { Discounts } from './Discounts';
+import { fakeAccountPlanDiscounts } from '../../../../services/doppler-account-plans-api-client.double';
+import { render, cleanup, waitFor, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import IntlProvider from '../../../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+
+const mockedhandleChange = jest.fn();
+
+const DiscountsElement = ({ discountsList, sessionPlan, selectedPlanDiscount, disabled }) => (
+  <IntlProvider>
+    <Discounts
+      discountsList={discountsList}
+      sessionPlan={sessionPlan}
+      selectedPlanDiscount={selectedPlanDiscount}
+      disabled={disabled}
+      handleChange={mockedhandleChange}
+    />
+  </IntlProvider>
+);
+
+describe('Discount component', () => {
+  afterEach(cleanup);
```

Comment:
It's not necessary. [More details](https://github.com/FromDoppler/doppler-webapp/pull/1504)

Reply:
👍